src/sentry/static/sentry/app/components/events/interfaces/openInContextLine.jsx

```diff
@@ -21,26 +21,37 @@ class OpenInContextLine extends React.Component {
     };
   }
 
-  getUrl() {
-    const {filename, lineNo, components} = this.props;
+  getUrl(component) {
+    const {filename, lineNo} = this.props;
 
     const queryParams = {
       lineNo,
       filename,
     };
-    return addQueryParamsToExistingUrl(components[0].schema.url, queryParams);
+    return addQueryParamsToExistingUrl(component.schema.url, queryParams);
   }
 
+  renderOneComponent = component => {
```

Comment:
We should call this something like `renderOpenInLink` to be more specific.

Reply:
@mnoble changed